velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -200,5 +200,38 @@ TEST_F(NthValueTest, offsetValues) {
       {vectors}, "nth_value(c0, c2)", overClause, offsetError);
 }
 
+TEST_F(NthValueTest, basicRangeFrames) {
```

Comment:
Yes, but now that the aggregate window function changes are merged, I will add the frame tests to the aggregate window function tests as well. Thanks for the suggestion! 
Should I add these tests for the ranking window functions too (since the frameBounds are not used in the call to apply in the ranking functions, hadn't added the tests there)?